src/CSSList/CSSList.php

```diff
@@ -296,6 +296,27 @@ public function splice($iOffset, $iLength = null, $mReplacement = null)
         array_splice($this->aContents, $iOffset, $iLength, $mReplacement);
     }
 
+    /**
+     * Insert an item before or after its sibling.
+     *
+     * @param RuleSet|CSSList|Import|Charset $oItem The item.
+     * @param RuleSet|CSSList|Import|Charset $oSibling The sibling.
+     * @param string $sPosition The position.
+     *
+     * @return void
+     */
+    public function insert(RuleSet|CSSList|Import|Charset $oItem, RuleSet|CSSList|Import|Charset $oSibling, string $sPosition = 'before'): void
```

Comment:
As we still support PHP 7.2, we cannot have native union types. So we'll need to omit the native type declarations for those and have them in PHPDoc annotations only instead.

Reply:
Yeah, figured that out. :/ It's fixed now :)